kong/init.lua

```diff
@@ -1155,4 +1159,38 @@ function Kong.serve_cluster_listener(options)
 end
 
 
+do
+  local declarative = require("kong.db.declarative")
+  local cjson = require("cjson.safe")
+
+  function Kong.stream_config_listener()
+    kong_global.set_phase(kong, PHASES.admin_api)
```

Comment:
This is a bit hacky but I do vaguely recall running into the same problem in any server block that uses PDK. Is it possible to avoid this in any way?

Reply:
This code path should't use PDK at all, but removing it shouldn't hurt either.

It was added only for styling reasons.